Assets/Scripts/PlayerSetup.cs

```diff
@@ -65,7 +67,19 @@ void DisableComponents()
 
     void AssignRemoteLayer()
     {
-        gameObject.layer = LayerMask.NameToLayer(_remoteLayerName);
+        RecursivelyAssignRemoteLayer(gameObject);
+    }
+
+    void RecursivelyAssignRemoteLayer(GameObject gObj)
```

Comment:
There is a method in Helpers that does this.  You might need  to change it to add the condition to it.

Reply:
This has now been addressed